### PR TITLE
styling improvements

### PR DESCRIPTION
**Pull request by holygits**
Screen Shot 2021-04-20 at 16 49 24

Screen Shot 2021-04-21 at 10 59 21

messaging above inputs
Screen Shot 2021-04-21 at 11 09 03

### CODE REVIEW

File: src/components/AccountPicker.tsx [hunk 1]
```diff
@@ -32,17 +46,18 @@ const AccountPicker: FC<AccountPickerProps> = ({title, selected, options, onChan
...
-            <Identicon value={selected} size={32} theme={'beachball'} />
+        <AccountWrapper className="accountWrapper">
+            <Identicon value={selected} size={32} theme={'beachball'} className="identicon" />
             <DropDown
                 className="select"
                 value={getOptionByValue(options, selected)}
                 options={options}
                 onChange={picked => onChange(picked)}
                 help={getOptionByValue(options, selected)}
+                disabled={true}
```
Comment (holygits) [inline src/components/AccountPicker.tsx:57]: need to remove this

File: static.config.js [hunk 2]
```diff
@@ -11,23 +11,14 @@ export default {
     getRoutes: async () => {
         try {
             return [
-                // A simple route
```
Comment (holygits) [inline static.config.js:14]: unused, I guess left over from static template

File: src/pages/liquidity/liquidity.tsx [hunk 3]
```diff
@@ -417,9 +318,8 @@ export const Liquidity: FC<LiquidityProps & LiquidityDispatchProps> = props => {
                                 assetDialogOpen: state.assetDialogOpen,
                             });
                         }}
-                        message=""
+                        message={`Public Address: ${signingAccount}`}
```
Comment (KarishmaBothara) [inline src/pages/liquidity/liquidity.tsx:321]: I think, here the signingAccount might not always exist... so we would need to check if it has value then show the string - Public Address: ${signingAccount} else no message?

File: src/pages/exchange/exchange.tsx [hunk 4]
```diff
@@ -175,9 +163,8 @@ export const Exchange: FC<ExchangeProps & ExchangeDispatchProps> = props => {
                             props.handleSelectedAccountChange(picked.value);
                             setState({touched: true, assetDialogOpen: state.assetDialogOpen});
                         }}
-                        message=""
+                        message={`Public Address: ${signingAccount}`}
```
Comment (KarishmaBothara) [inline src/pages/exchange/exchange.tsx:166]: same here

**Review by KarishmaBothara — APPROVED** [review]
lgtm..

**Pull request merged**